UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/Extensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Net.Http.Headers;
+
+namespace UKHO.PeriodicOutputService.Common
+{
+    [ExcludeFromCodeCoverage]
+    public static class HttpRequestMessageExtensions
```

Comment:
Please rename the file name similar to class name.

Reply:
renamed